tests/utils/test_fmrisim_real_time.py

```diff
@@ -0,0 +1,172 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""fmrisim real-time generator test script
+
+ Authors: Cameron Ellis (Princeton) 2020
+"""
+import numpy as np
+from brainiak.utils import fmrisim_real_time_generator as gen
+import pytest
+import os
+import nibabel as nib
+import time
+import glob
+import copy
+from pkg_resources import resource_stream
+
+# Test that it crashes without inputs
+with pytest.raises(TypeError):
+    gen.generate_data()
+
+data_dict = {}
+data_dict['ROI_A_file'] = resource_stream(__name__, "ROI_A.nii.gz")
```

Comment:
Change everywhere in tests to:
```python
resource_stream(fmrisim_real_time_generator.__name__, "ROI_A.nii.gz")
```

Reply:
This seems to be the cause of the error. Note that in the setup file I have the following:

```
    package_data={'brainiak.utils.sim_parameters': ['grey_matter_mask.npy',
                                                    'ROI_A.nii.gz',
                                                    'ROI_B.nii.gz',
                                                    'mask.npy',
                                                    'sub_noise_dict.txt',
                                                    'sub_template.nii.gz']},
```

I will also need to make this change for the base fmrisim code

Reply:
The "base fmrisim code" is in the same directory as the data file, so can just use `__name__` there.

Reply:
I am not sure I understand. I moved the 'grey_matter_mask.npy' file that `fmrisim.py` is referencing into this same directory that `fmrisim_real_time_generator.py` is referencing other data in.

Reply:
Sorry, I missed that move. You should use the following in the test code:
```python
resource_stream(fmrisim_real_time_generator.__name__, "sim_parameters/ROI_A.nii.gz")
```
And the following the brainiak.utils code:
```python
resource_stream(__name__, "sim_parameters/grey_matter_mask.npy")
```